core/entities/Module.php

```diff
@@ -308,7 +308,7 @@ public function saveSetting($setting, $value, $uid = 0, $scope = null)
             return framework\Settings::saveSetting($setting, $value, $this->getName(), $scope, $uid);
         }
 
-        public function deleteSetting($setting, $uid = null, $scope = null)
+        public function deleteSetting($setting, $uid = 0, $scope = null)
```

Comment:
A modification to the api (eg. function parameters) is exactly the case why I've made the first comment. You did change it to `0` by default right? ;) That's why I wanted to double-check it with you. I'm not planning on changing this ;)